tasks/main.yml

```diff
@@ -18,9 +7,16 @@
 
 - import_tasks: install.yml
   become: true
+  when: (not __bind_exporter_is_installed.stat.exists) or (__bind_exporter_current_version_output.stderr_lines[0].split(" ")[2] != bind_exporter_version)
```

Comment:
Apparently this is a problematic bit, which causes failures in CI. This is due to:
```
list object has no element 0
```
Since there is no bind_exporter when doing a clean install, there is also no content in `__bind_exporter_current_version_output.stderr_lines`.

Reply:
Hrm, how is this working in other roles?

Reply:
It turns out that the bind_exporter reports the version info on stdout, not stderr. :face_with_head_bandage: